languages/clojure/code/.codecrafters/run.sh

```
@@ -0,0 +1,11 @@
+#!/bin/sh
+#
+# This script is used to run your program on CodeCrafters
+#
+# This runs after .codecrafters/compile.sh
+#
+# Learn more: https://codecrafters.io/program-interface
+
+set -e # Exit on failure
+
+exec java -jar /tmp/codecrafters-build-{{course-slug}}-clojure/target.jar "$@"
```

Shellcheck (0.10.0)

[warning] This { is literal. Check expression (missing ;/\n?) or quote it.

(SC1083)